client/web/src/marketing/SurveyToast.tsx

```diff
@@ -1,6 +1,7 @@
 import EmoticonIcon from 'mdi-react/EmoticonIcon'
-import * as React from 'react'
-import { Link } from 'react-router-dom'
+import React, { useState } from 'react'
+import { useHistory } from 'react-router'
+import cln from 'classnames'
```

Comment:
nit: change to classNames for better readability and to align with how we typically import this

```suggestion
import classNames from 'classnames'
```